dht/CMakeLists.txt

```diff
@@ -19,3 +19,6 @@ target_link_libraries(scylla_dht
     replica)
 
 add_whole_archive(dht scylla_dht)
+
+check_headers(check-headers scylla_dht
+  GLOB ${CMAKE_CURRENT_SOURCE_DIR}/*.hh)
```

Comment:
Better to use GLOB_RECURSE everywhere, in case someone adds a subdirectory.

Reply:
ack. changed and repushed.